thirdeye/thirdeye-frontend/app/styles/pods/rca.scss

```diff
@@ -147,11 +147,15 @@
   }
 }
 
-.color-box {
+.container-fluid {
```

Comment:
let's not override bootstrap's classes directly. 
Please create a new class (we may use !important as long as we comment that it's to override bootstrap) 

Also, I feel this is not needed anymore after my legend pr got merged

Reply:
fixed